src/Symfony/Component/Security/composer.json

```diff
@@ -39,11 +39,6 @@
         "ircmaxell/password-compat": "~1.0",
         "symfony/expression-language": "~2.6"
     },
-    "replace": {
-      "symfony/security-acl": "self.version",
```

Comment:
these lines are duplicates: a replace section already exists a few lines above in the file